src/main/java/frc/robot/RobotContainer.java

```diff
@@ -227,105 +228,71 @@ private void configureBindings() {
         if (FeatureFlags.runDrive) {
             setUpDriveWithJoysticks();
                 
-            rightJoystick.trigger()
+            leftJoystick.trigger()
                 .onTrue(new InstantCommand(
                     () -> drivetrain.setAlignState(AlignState.ALIGNING)))
                 .onFalse(new InstantCommand(
                     () -> drivetrain.setAlignState(AlignState.MANUAL)));
 
             leftJoystick.top()
                 .onTrue(new InstantCommand(
-                    () -> drivetrain.seedFieldRelative(getPoseAgainstSpeaker()))
-                );
-
-            controller.b()
+                    () -> drivetrain.seedFieldRelative(getPoseAgainstSpeaker())));
+            
+            leftJoystick.button(1) // placeholder button
                 .onTrue(new InstantCommand(
-                    () -> drivetrain.setAlignTarget(AlignTarget.SPEAKER)));
+                    () -> drivetrain.seedFieldRelative(getPoseAgainstPodium())));
 
-            controller.start()
+            controller.povUp()
                 .onTrue(new InstantCommand(
                     () -> drivetrain.setAlignTarget(AlignTarget.SPEAKER)));
 
-            controller.x()
+            controller.povRight()
                 .onTrue(new InstantCommand(
-                    () -> drivetrain.setAlignTarget(AlignTarget.SPEAKER)));
+                    () -> drivetrain.setAlignTarget(AlignTarget.AMP)));
 
-            controller.back()
+            controller.povLeft()
                 .onTrue(new InstantCommand(
-                    () -> drivetrain.setAlignTarget(AlignTarget.AMP)));
+                    () -> drivetrain.setAlignTarget(AlignTarget.SOURCE)));
 
-            // controller.povRight().onTrue(drivetrain.getDriveToPointCommand());
+            controller.povDown()
+                .onTrue(new InstantCommand(
+                    () -> drivetrain.setAlignTarget(AlignTarget.ENDGAME)));
```

Comment:
If we have these here instead of in DriveWithJoysticks, will we interrupt driving for a frame because there will be 2 commands that require the drivetrain? Or will the Scheduler handle this well?

Reply:
No. The instant commands would have to require the drive explicitly.